hack/deploy-logging.sh

```diff
@@ -331,14 +329,10 @@ deploy_logging_using_olm() {
 
 deploy_logging_using_clo_make() {
     # edit the deployment manifest - use the images provided by CI or from api.ci registry
-    # make deploy-no-build
-    EO_IMAGE=${EO_IMAGE:-$( construct_image_name elasticsearch-operator latest )}
-    CLO_IMAGE=${CLO_IMAGE:-$( construct_image_name cluster-logging-operator latest )}
     pushd $CLO_DIR > /dev/null
     cp manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml.orig
-    update_images_in_clo_yaml manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml $CLO_IMAGE
-    REMOTE_CLUSTER=true REMOTE_REGISTRY=true NAMESPACE=$LOGGING_NS \
-        IMAGE_OVERRIDE="$CLO_IMAGE" EO_IMAGE_OVERRIDE="$EO_IMAGE" make deploy-no-build
+    update_images_in_clo_yaml manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml 
+    make deploy
```

Comment:
we can capture the `$?` value and throw error in case we ever remove/change `make deploy` target in CLO Makefile

Reply:
wont we know this by the fact the operation fails?